src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurlySwitchCasesBlocks.java

```diff
@@ -95,13 +95,11 @@ public static void test4() {
       case 1: { // violation ''{' at column 15 should be on a new line.'
           int x = 1; // violation '.* incorrect indentation level 10, expected level should be 8.'
           break; // violation '.* incorrect indentation level 10, expected level should be 8.'
-          // 2 violations 3 lines below:
-          //  ''block rcurly' has incorrect indentation level 8, expected level should be 6.'
-          //  ''}' at column 9 should be alone on a line.'
+          // violation below ''}' at column 9 should be alone on a line.'
         } default:
-          { // violation '.* incorrect indentation level 10, expected level should be 8.'
+          {
                   int x = 0; // violation '.* incorrect indentation .*, expected .* 8, 10.'
-          } // violation '.* incorrect indentation level 10, expected level should be 8.'
+          }
```

Comment:
I did not expect change in behavior in this file.
Please share AST for case/default, we might need to adjust xpath a bit to not suppress such cases.

Reply:
AST of:

```java
public class Test {
  public static void test4() {
    int mode = 0;
    switch (mode) {
      case 1: {
          int x = 1;
          break;
        } default:
          {
                  int x = 0;
          }
    }
  }
}
```

```
$ java -jar .\checkstyle-10.18.2-all.jar -t .\Test.java
COMPILATION_UNIT -> COMPILATION_UNIT [1:0]
`--CLASS_DEF -> CLASS_DEF [1:0]
    |--MODIFIERS -> MODIFIERS [1:0]
    |   `--LITERAL_PUBLIC -> public [1:0]
    |--LITERAL_CLASS -> class [1:7]
    |--IDENT -> Test [1:13]
    `--OBJBLOCK -> OBJBLOCK [1:18]
        |--LCURLY -> { [1:18]
        |--METHOD_DEF -> METHOD_DEF [2:2]
        |   |--MODIFIERS -> MODIFIERS [2:2]
        |   |   |--LITERAL_PUBLIC -> public [2:2]
        |   |   `--LITERAL_STATIC -> static [2:9]
        |   |--TYPE -> TYPE [2:16]
        |   |   `--LITERAL_VOID -> void [2:16]
        |   |--IDENT -> test4 [2:21]
        |   |--LPAREN -> ( [2:26]
        |   |--PARAMETERS -> PARAMETERS [2:27]
        |   |--RPAREN -> ) [2:27]
        |   `--SLIST -> { [2:29]
        |       |--VARIABLE_DEF -> VARIABLE_DEF [3:4]
        |       |   |--MODIFIERS -> MODIFIERS [3:4]
        |       |   |--TYPE -> TYPE [3:4]
        |       |   |   `--LITERAL_INT -> int [3:4]
        |       |   |--IDENT -> mode [3:8]
        |       |   `--ASSIGN -> = [3:13]
        |       |       `--EXPR -> EXPR [3:15]
        |       |           `--NUM_INT -> 0 [3:15]
        |       |--SEMI -> ; [3:16]
        |       |--LITERAL_SWITCH -> switch [4:4]
        |       |   |--LPAREN -> ( [4:11]
        |       |   |--EXPR -> EXPR [4:12]
        |       |   |   `--IDENT -> mode [4:12]
        |       |   |--RPAREN -> ) [4:16]
        |       |   |--LCURLY -> { [4:18]
        |       |   |--CASE_GROUP -> CASE_GROUP [5:6]
        |       |   |   |--LITERAL_CASE -> case [5:6]
        |       |   |   |   |--EXPR -> EXPR [5:11]
        |       |   |   |   |   `--NUM_INT -> 1 [5:11]
        |       |   |   |   `--COLON -> : [5:12]
        |       |   |   `--SLIST -> SLIST [5:14]
        |       |   |       `--SLIST -> { [5:14]
        |       |   |           |--VARIABLE_DEF -> VARIABLE_DEF [6:10]
        |       |   |           |   |--MODIFIERS -> MODIFIERS [6:10]
        |       |   |           |   |--TYPE -> TYPE [6:10]
        |       |   |           |   |   `--LITERAL_INT -> int [6:10]
        |       |   |           |   |--IDENT -> x [6:14]
        |       |   |           |   `--ASSIGN -> = [6:16]
        |       |   |           |       `--EXPR -> EXPR [6:18]
        |       |   |           |           `--NUM_INT -> 1 [6:18]
        |       |   |           |--SEMI -> ; [6:19]
        |       |   |           |--LITERAL_BREAK -> break [7:10]
        |       |   |           |   `--SEMI -> ; [7:15]
        |       |   |           `--RCURLY -> } [8:8]
        |       |   |--CASE_GROUP -> CASE_GROUP [8:10]
        |       |   |   |--LITERAL_DEFAULT -> default [8:10]
        |       |   |   |   `--COLON -> : [8:17]
        |       |   |   `--SLIST -> SLIST [9:10]
        |       |   |       `--SLIST -> { [9:10]
        |       |   |           |--VARIABLE_DEF -> VARIABLE_DEF [10:18]
        |       |   |           |   |--MODIFIERS -> MODIFIERS [10:18]
        |       |   |           |   |--TYPE -> TYPE [10:18]
        |       |   |           |   |   `--LITERAL_INT -> int [10:18]
        |       |   |           |   |--IDENT -> x [10:22]
        |       |   |           |   `--ASSIGN -> = [10:24]
        |       |   |           |       `--EXPR -> EXPR [10:26]
        |       |   |           |           `--NUM_INT -> 0 [10:26]
        |       |   |           |--SEMI -> ; [10:27]
        |       |   |           `--RCURLY -> } [11:10]
        |       |   `--RCURLY -> } [12:4]
        |       `--RCURLY -> } [13:2]
        `--RCURLY -> } [14:0]
```

The violation was a false-positive so I think this is good that we were able to suppress it. I guess there's no need to change xpath